presto-main/src/main/java/io/prestosql/execution/QueryStateTimer.java

```diff
@@ -104,8 +111,10 @@ public void beginStarting()
 
     private void beginStarting(long now)
```

Comment:
make this:
```
private void beginStarting(long cpuNow)
```
for consistency

Reply:
@sopel39 Thank you for your comment.

Do you mean `private void beginStarting(long now, long cpuNow)` instead of `private void beginStarting(long cpuNow)`? If we change the `beginStarting` as well, should we change `beginRunning` as well? Thanks!

Reply:
`private void beginStarting(long now, long cpuNow) `

Yes, this one. Thanks

Why need to change `beginRunning`?

Reply:
Ahh, I know what you mean now, addressed! Thanks!